pyvo/mivot/seekers/annotation_seeker.py

```diff
@@ -0,0 +1,321 @@
+"""
+Created on 11 Dec 2021
+
+@author: laurentmichel
```

Comment:
normally we don't keep author info in the files (it's all captured in the gitlog), but instead keep the one line licence info in the first line. See examples in the other files.